tests/src/Unit/ReadonlyWidgetTest.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Drupal\Tests\readonly_field_widget\Unit;
+
+use Drupal\Tests\UnitTestCase;
+use Drupal\readonly_field_widget\Plugin\Field\FieldWidget\ReadonlyWidget;
+
+class ReadonlyWidgetTest extends UnitTestCase {
```

Comment:
All Drupal tests should have an `@group` annotation in the class comment, typically with the module name. Otherwise, `run-tests.sh` (if it's used) can fail or miss tests.